src/components/POLine/POLineDetails/FieldSource.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'redux-form';
+
+import { Select } from '@folio/stripes/components';
+
+function FieldSource({ sources = [] }) {
```

Comment:
maybe, it's better to use arrow function for stateless components

Reply:
In this case there is no difference, lets wait common eslint config